templates/flamegraph.mustache

```diff
@@ -172,8 +172,8 @@
 <div id="details" style="min-height: 1.5rem; clear: both;">
 </div>
 
-<script type="text/javascript" src="https://d3js.org/d3.v4.min.js"></script>
-<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/d3-flame-graph@4.0.6/dist/d3-flamegraph.min.js"></script>
+<script type="text/javascript" src="/admin/tool/excimer/lib/d3/d3.min.js"></script>
```

Comment:
It's better to route the serving of these through /theme/javascript.php which will add all the proper cache busting params and caching headers to it gets cached by cdn / varnish

Reply:
How does this work?